src/Surfnet/StepupRa/RaBundle/DependencyInjection/SurfnetStepupRaRaExtension.php

```diff
@@ -50,5 +50,7 @@ public function load(array $configs, ContainerBuilder $container)
             'ra.security.authentication.session.maximum_relative_lifetime_in_seconds',
             $config['session_lifetimes']['max_relative_lifetime']
         );
+
+        $container->setParameter('surfnet_stepup_ra.self_service_url', $config['self_service_url']);
```

Comment:
This could be a replaceParameter as well, but I chose this to be more in line with what we already do. Recommendations welcome.